### DISCUSSION TITLE
Automated Ticket Creation For DQ Failures

### DISCUSSION THREAD

**Discussion by jmcorreia** [great-expectations/great_expectations#7162] [Ideas]
Hi all :)

We have been using Great Expectations for checking the Quality of our Data and we have use cases for which we would like to have an automated way to create tickets or integrate with other platforms that give us this possibility and observability/visibility. 
These kind of integrations is something that tools like Monte Carlo enable, so maybe we can get some ideas there. 

GX could have ways to integrate with third party tools (e.g. jira, opsgenie) and configure/create rules (e.g. we probably don't want a ticket for each single failure on a expectation) that would automatically create tickets.

Is this something that GX will consider in the Roadmap? What would be your thoughts? 

Thanks!

**Comment by lodsdevera** [comment]
Thought this might be related to this since opsgenie was mentioned
[https://docs.greatexpectations.io/docs/guides/validation/validation_actions/how_to_trigger_opsgenie_notifications_as_a_validation_action](url)

My simple mind thought that the configure/create rules thingy can be on the third party tool's part by utilising the notification that was sent and use it as the trigger to do the rule.

P.S. Don't have any experience yet with opsgenie, just sharing some quick thoughts

**Reply by jmcorreia** [reply]
Nice one, missed that link. I will still keep the discussion for alternatives and for the rules part in case someone has ideas around it.

**Reply by lodsdevera** [reply]
Yeah looking forward for some ideas around the configure/rules part too.

**Comment by a-chumagin** [comment]
@jmcorreia in our Data Quality Gate we have integration with Jira
https://github.com/provectus/data-quality-gate/blob/main/functions/report_push/jira_events.py

look please maybe it is will work for you
